Datastore/src/V1/Gapic/DatastoreGapicClient.php

```diff
@@ -228,6 +228,7 @@ public function allocateIds($projectId, $keys, array $optionalArgs = [])
         $requestParamHeaders['project_id'] = $projectId;
         if (isset($optionalArgs['databaseId'])) {
             $request->setDatabaseId($optionalArgs['databaseId']);
+            $requestParamHeaders['database_id'] = $optionalArgs['database_id'];
```

Comment:
Aha! We found a bug in the generator. The `isset` is using camel-cased `databaseId`, but then switches to snake-cased `database_id` when accessing the array.

Reply:
Fixed in https://github.com/googleapis/gapic-generator-php/pull/574